### DISCUSSION TITLE
Can't seem to clone the nouns builder repository.

### DISCUSSION THREAD

**Discussion by fractalqualia** [ourzora/nouns-builder#342] [General]
Hello 👋🏻, I'm trying to clone the nouns builder frontend repo to contribute.
However, I am running into an issue and can't get the dev environment to run

I have gotten Etherscan API key and The alchemy API key

One question I have is are `PRIVATE_ALCHEMY_ID` in the `.env file `(root dir) and the `NEXT_PUBLIC_ALCHEMY_ID` in the `.env.local` (in apps/web folder) file the same thing?

I have set both APIs and tried to run it : 
`pnpm dev`

 I get the following
![image](https://github.com/ourzora/nouns-builder/assets/113944109/10850896-6acf-4eb0-841e-a837e3309939)

But when i open the webpage i get a 401 error. 

Thank you.

**Comment by isaaczaak** [comment]
Thanks for reaching out! Could we help @fractalqualia out here, @neokry ?
